srcpkgs/sirikali/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'sirikali'
+pkgname=sirikali
+version=1.5.0
+revision=1
+#archs="i686 x86_64"
+wrksrc="SiriKali-${version}"
+create_wrksrc=yes
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DQT5=true
+ -DCMAKE_BUILD_TYPE=RELEASE"
```

Comment:
```suggestion
configure_args="-DQT5=true"
```
the build style defines two of those already:
https://github.com/void-linux/void-packages/blob/69267e8b403c0b6b29732fb3aef96ad0b4876547/common/build-style/cmake.sh#L54-L55